src/samples/GroupChannelFreezeUnfreeze.js

```diff
@@ -0,0 +1,621 @@
+
+            <div className="message-input-buttons">
+                <button className="send-message-button" onClick={sendMessage}>Send Message</button>
+                <label className="file-upload-label" htmlFor="upload" >Select File</label>
+
+                <input
+                    id="upload"
+                    className="file-upload-button"
+                    type='file'
+                    hidden={true}
+                    onChange={onFileInputChange}
+                    onClick={() => { }}
+                />
+            </div>
+
+        </div>);
+}
+
+const MembersSelect = ({
+    applicationUsers,
+    groupChannelMembers,
+    currentlyJoinedChannel,
+    addToChannelMembersList,
+    handleCreateChannel,
+    handleUpdateChannelMembersList
+
+}) => {
+
+    if (applicationUsers.length > 0) {
+        return <div className="overlay">
+            <div className="overlay-content">
+                <button onClick={() => {
+                    if (currentlyJoinedChannel) {
+                        handleUpdateChannelMembersList();
+                    } else {
+                        handleCreateChannel();
+
+                    }
+                }}>{currentlyJoinedChannel ? 'Submit' : 'Create'}</button>
+                {applicationUsers.map((user) => {
+                    const userSelected = groupChannelMembers.some((member) => member === user.userId);
+                    return <div
+                        key={user.userId}
+                        className={`member-item ${userSelected ? 'member-selected' : ''}`}
+                        onClick={() => addToChannelMembersList(user.userId)}>
+                        <ProfileImage user={user} />
+                        <div className="member-item-name">{user.nickname}</div>
+                    </div>
+                })}
+
+            </div>
+        </div >;
+    }
+    return null;
+}
+
+const CreateUserForm = ({
+    setupUser,
+    settingUpUser,
+    userNameInputValue,
+    userIdInputValue,
+    onUserNameInputChange,
+    onUserIdInputChange
+}) => {
+    if (settingUpUser) {
+        return <div className="overlay">
+            <div className="overlay-content">
+                <div>User ID</div>
+
+                <input
+                    onChange={onUserIdInputChange}
+                    className="form-input"
+                    type="text" value={userIdInputValue} />
+
+
+                <div>User Nickname</div>
+                <input
+                    onChange={onUserNameInputChange}
+                    className="form-input"
+                    type="text" value={userNameInputValue} />
+
+                <button
+                    className="user-submit-button"
+                    onClick={setupUser}>Connect</button>
+            </div>
+        </div>
+
+
+    } else {
+        return null;
+    }
+
+}
+
+const DisplayFreezeMessage = () => (
+  <div className='frozen-toast'>Channel Frozen </div>
```

Comment:
Small one but i think alot of devs would be confused by the toast concpet here. Can we make more generic? "notification" or "banner" or "alert"